src/main/java/com/example/bookstoreapp/entities/OrderEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.example.bookstoreapp.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "orders")
+@Data
+@NoArgsConstructor
+public class OrderEntity {
+    @Id
+    private long order_id;
+    private long customer_id;
```

Comment:
For java entity class here instead customer_id use customerId. In database it will be customer_id.
Same applies for other entity class fields.